devel/SourceKitten/Portfile

```
@@ -5,7 +5,7 @@ PortGroup           github 1.0
 PortGroup           xcodeversion 1.0
 
 use_xcode           yes
-github.setup        jpsim SourceKitten 0.31.0
+github.setup        jpsim SourceKitten 0.32.0
 categories          devel
 platforms           darwin
```

Although the use of MacPorts on pure Darwin (and the use of pure Darwin in general) is hypothetical, I assume that SourceKitten would not run on it since it uses Apple's new Combine framework, which I assume is closed-source based on the existence of an [open-source reimplementation of it](https://github.com/OpenCombine/OpenCombine). Therefore, change the (cosmetic) indicator of supported platforms to exclude pure Darwin:
```suggestion
platforms           macosx
```

---

I'm only a casual user of SourceKitten but according to https://github.com/jpsim/SourceKitten it runs on Linux in addition to macOS. I don't believe it requires the Combine framework.